tests/acceptance/features/cliProvisioning/userSync.feature

```diff
@@ -38,4 +38,20 @@ Feature: sync a user using occ command
       """
       OC\User\Database
       OCA\User_LDAP\User_Proxy
-      """
\ No newline at end of file
+      """
+
+  Scenario: admin deletes ldap users and performs full sync
+    When the administrator deletes user "user0" using the occ command
+    And the administrator deletes user "user1" using the occ command
+    Then user "user0" should not exist
+    And user "user1" should not exist
+    When the LDAP users have been resynced
+    Then user "user0" should exist
+    And user "user1" should exist
+
+  Scenario: sync a local user
+    Given user "local-user" has been created with default attributes in the database user backend
+    When the administrator changes the display name of user "local-user" to "Test User" using the occ command
+    And LDAP user "local-user" is resynced
+    Then the command should have been successful
+    And user "local-user" should not exist
```

Comment:
This might be surprising for an admin. The command deleted a local user.
I will make an issue to discuss that

Reply:
OK, I'll add an issue tag!

Reply:
No need for an issue tag - we are not sure yet if it is really an issue to be fixed.